scalalib/test/src/mill/scalalib/HelloWorldTests.scala

```diff
@@ -45,11 +48,79 @@ object HelloWorldTests extends TestSuite {
   }
 
   object HelloWorldWithMain extends HelloBase {
-    object core extends HelloWorldModule{
-      def mainClass = Some("Main")
+    object core extends HelloWorldModuleWithMain
+  }
+
+  val akkaHttpDeps = Agg(ivy"com.typesafe.akka::akka-http:10.0.13")
```

Comment:
We really need to break up the tests in this file :/ Maybe not in this PR, but if you could do that in a subsequent one that would be great

Reply:
Agreed